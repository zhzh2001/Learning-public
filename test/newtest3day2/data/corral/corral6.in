19 50
222143788913334243 76132217705922532
617527890950266363 232046586796457967
597712576946863935 339159054469216072
840274771118703982 50596646150009007
143076682877468113 636613349226385022
671588129753619356 569810796091527503
214487396477783233 466075444867711060
331661358163660178 932351875768679999
199831955323805931 151230294695219816
583991373030434953 763958345030572768
887222166551271030 579596433912614311
8140117964808842 961321382709924670
941151253178546344 239089061538606213
69713776214816704 820870258280419904
969681025453379938 685055085253693377
511655970763650989 923973191723095197
222276399361766779 681569502538117838
901890122217457500 624957409257449158
784704365490834582 465152596760151555
996858605414069608 224709420737162505
468101026192119157 906665197971997282
511479092401880443 784471091285569613
801642099770735982 994313563367438022
414527109459142295 919304769595739547
328497089366363102 335946911809399587
510595463019377093 148089789941085001
899889088838411199 371390371242480020
869923448616163068 376770548477301103
425453402025533742 151269255699164912
555052167660608289 779637972419435227
477896423056411897 475374837935025370
547177382271854480 668037831094559478
282121164944065744 326177993651704100
711753187789890238 132607650019942438
305126337313582448 431450284638491025
34835147175749885 587394927380260810
654481929562036716 120188646945592566
502496425815349315 642093060540319912
900088936603002178 750924825060915920
60447210386851423 397018518404709069
739353501408092846 728930330039894028
994854320452405816 443341094536212623
841977646830457389 937242762843980633
685491939885561253 441303194138322601
775007628600936914 584407416674547774
915189024159782441 171367453092859932
768121330764413932 669409485655182391
281595250452882438 202906049617516436
395699643000208613 613418801164592069
60940639119623666 354923732004987323
