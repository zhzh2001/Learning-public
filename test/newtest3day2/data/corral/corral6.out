491784602396968042
